Add Royal Danish osu!Cup (#3087)
ppy/osu-wiki: wiki/Tournaments/RDOC/RDOC_2018/en.md

---
tags:
  - RDOC
  - RDOC2018
---

# Royal Danish osu!Cup 2018

> ![RDOC Logo](img/logo.jpg)

Reviewer: ```suggestion
![RDOC logo](img/logo.jpg)
```
Sentence case.

> The **Royal Danish osu!Cup 2018** (***RDOC2018***) was a national double-elimination 32-player 1v1 osu!standard tournament hosted by ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742), ![][flag_DK] [Gladand](https://osu.ppy.sh/users/8996506)  and ![][flag_DK] [Crylizhy](https://osu.ppy.sh/users/3023138). This was the fourth installment of the Royal Danish osu!Cup.

Reviewer: ```suggestion
The **Royal Danish osu!Cup 2018** (***RDOC2018***) was a Danish national double-elimination 32-player 1v1 osu!standard tournament hosted by ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742), ![][flag_DK] [Gladand](https://osu.ppy.sh/users/8996506), and ![][flag_DK] [Crylizhy](https://osu.ppy.sh/users/3023138). It was the fourth installment of the Royal Danish osu!Cup.
```
Idk, sounds better.
Either Danish national or just Danish

## Tournament schedule

| Event | Timestamp |
| --: | :-- |

> | Registration Phase | 2018-08-13/2018-09-07 |

Reviewer: ```suggestion
| Registration phase | 2018-08-13/2018-09-07 |
```

> | Live Drawings | 2018-09-09 |

Reviewer: ```suggestion
| Live drawings | 2018-09-09 |
```

| Group Stage | 2018-09-15/2018-09-16 & 2018-09-22/2018-09-23 |
| Round of 16 | 2018-09-29/2018-09-30 |
| Quarterfinals | 2018-10-06/2018-10-07 |
| Semifinals | 2018-10-13/2018-10-14 |
| Finals | 2018-10-20/2018-10-21 |
| Grand Finals | 2018-10-27/2020-10-28 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, 4 months of osu!supporter tag |
| ![Silver crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
| ![Bronze crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |

> ![RDOC Winner Badge](img/badge.png)

Reviewer: ```suggestion
![RDOC winner badge](img/badge.png)
```

## Organisation

> The Royal Danish osu!Cup is run by various community members.

Reviewer: ```suggestion
The Royal Danish osu!Cup 2018 was run by various community members.
```

| Low | ![][flag_DK] [Lucasion](https://osu.ppy.sh/users/7382672), ![][flag_DK] [Boje](https://osu.ppy.sh/users/5120773), ![][flag_DK] [Power Supply](https://osu.ppy.sh/users/7091513), ![][flag_DK] [_Alex](https://osu.ppy.sh/users/7882154), ![][flag_DK] [Pallerik](https://osu.ppy.sh/users/9303652), ![][flag_DK] [P3RS3X](https://osu.ppy.sh/users/9469362), ![][flag_DK] [eLLezH](https://osu.ppy.sh/users/2143311), ![][flag_DK] [\[THEDUCK\]](https://osu.ppy.sh/users/6080866) |
| Bottom | ![][flag_DK] [Ghost Krillin](https://osu.ppy.sh/users/6913805), ![][flag_DK] [FeelDaTonneh](https://osu.ppy.sh/users/4348782), ![][flag_DK] [Kinety](https://osu.ppy.sh/users/5941273), ![][flag_DK] [Waven7](https://osu.ppy.sh/users/11157036), ![][flag_DK] [seb2110](https://osu.ppy.sh/users/5323224), ![][flag_DK] [Syncopate](https://osu.ppy.sh/users/7232730), ![][flag_DK] [LeDoodeNiss](https://osu.ppy.sh/users/7345263), ![][flag_DK] [sanoJ](https://osu.ppy.sh/users/5385297) |

## Groups

> | Group | Top Seed | High Seed | Low Seed | Bottom Seed |

Reviewer: ```suggestion
| Group | Top seed | High seed | Low seed | Bottom seed |
```

- HardRock
  - [LeaF - Wizdomiot \[Another\]](https://osu.ppy.sh/beatmapsets/336099#osu/754799)
  - [Nanahoshi Kangengakudan - IMAGINARY LIKE THE JUSTICE \[Frostings' Insane\]](https://osu.ppy.sh/beatmapsets/210346#osu/508558)
  - [Xi - Majotachi no Butoukai ~ Magus \[Extra Stage\]](https://osu.ppy.sh/beatmapsets/43862#osu/158309)
- DoubleTime
  - [3L - Macrophylla Parasol \[Lunatic\]](https://osu.ppy.sh/beatmapsets/483344#osu/1489084)
  - [Giga-P - Electric Angel \[Insane\]](https://osu.ppy.sh/beatmapsets/146794#osu/363639)
  - [Zektbach - ZETA \~Sosuu no Sekai to Chouetsu sha\~ \[Hard\]](https://osu.ppy.sh/beatmapsets/34349#osu/111685)
- FreeMod
  - [Hatsuki Yura - Shoujo to Ougonryuu no Monogatari \[Collab Insane\]](https://osu.ppy.sh/beatmapsets/719158#osu/1542952)
  - [Rise Against - The Eco-Terrorist In Me \[Expert\]](https://osu.ppy.sh/beatmapsets/358609#osu/819082)
  - [kors k feat. Rie - Rising in the Sun(original mix) \[Another\]](https://osu.ppy.sh/beatmapsets/26742#osu/90024)
- Tiebreaker
  - **[An - Dive Archive \[Inundation\]](https://osu.ppy.sh/beatmapsets/761883#osu/1601943)**

  ------------------------------------------------------------------------

>   ## Match Results

Reviewer: ```suggestion
  ## Match results
```

  ### Grand Finals

>   | Group | Sunday, 2018-10-28 |  |  |  |

Reviewer: Use this formatting instead.

Sunday, 28 October 2018:

| Group | Player 1 |  |  | Player 2 | Match link (if there are any, don't include if it doesn't have all the records) |
| :-: | --: | :-: | :-: | :-- | :-- |
